docs/roadmap.md

```diff
@@ -23,7 +23,7 @@ This roadmap is a constant work in progress, subject to frequent revision. Dates
 - [AWS Fault injector integration to improve resiliency](https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/2173)
 - AWSMachinePool implementation backed by Spot Fleet and EC2 Fleet
 - [Dual stack IPv4/IPv6 support](https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/3381)
-- Windows nodes
+- [Windows Worker Node Support for Windows Server 2019/2022 for both Self Managed and EKS clusters](https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/3533)
```

Comment:
```suggestion
- [Windows Worker Node Support for Windows Server 2019/2022 for both CAPA-managed and EKS-managed clusters](https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/3533)
```

Reply:
tyvm, fixing

Reply:
Self managed has a different meaning in the cluster api ecosystem (means management cluster with controllers manage itself).

Reply:
you know what never mind, i read what you said and i get it now... my feedback then is maybe the CAPA-managed and EKS-managed verbiage should be in the About or Readme with descriptions since they're such different concepts.